packages/mobile/src/components/offline-downloads/OfflineDownloader.tsx

```diff
@@ -29,6 +29,7 @@ export const OfflineDownloader = () => {
 
   const isReachable = useSelector(getIsReachable)
 
+  // Move this logic to watch set (un)rechable sagas?:
```

Comment:
Part of the reason this happened was because this "go offline effect" logic existed in two separate places, here and the sagas file. So I think I'm gonna move the logic in this effect to the handleSet(Un)reachable sagas unless y'all think it works better here!